content/articles/web-apps.md

```diff
@@ -0,0 +1,108 @@
+---
+author: "Emma Russell"
+date: 2023-09-12
+title: "RESIDE-IC Web Applications"
+tags:
+ - web
+---
+
+# Introduction
+
+This page describes some aspects to think about when considering a new RESIDE web application to present research:
+
+- why a research team might want a web application
+- whether a Shiny app or full web application would better meet their needs
+- the components of a full research web application
+- what we require from research in order to build an interface to it which a new web application can use
+- some different patterns of application we can provide
+
+# Why have a web application?
+..as opposed to making research code available for users to download and run themselves?
+
+Web apps are:
+- Accessible - no special skills required from user, independent of their coding knowledge
+- Independent of user's local system, available anywhere. Can deploy to appropriate compute resource.
+- Can deploy known stable versions - this helps with reproducible research
+- Usability - can design the user experience to focus on their concerns e.g. particular practical/policy goals, or a need to explore a research model in a guided way with appropriate visualisations.
+- Can provide a user model allowing users to easily save, manage and share their data
+- User does not need direct access to large datasets required for code execution
+- Can give researchers quick means to assess utility of a model before putting in effort to download, configure and run locally.
```

Comment:
This list felt a bit weird to read to me especially parts where it would read "web apps are can deploy known stable versions". Obviously feel free to take some or none of the suggestions but this is how i would write it
```suggestion
Web apps are:
- Accessible - no special skills required from user, independent of their coding knowledge
- Available on all devices, local systems and anywhere in the world
- Flexible - can deploy to appropriate compute resource or deploy known stable versions, helping with reproducible research
- Adaptable - can tailor the user experience to focus on their concerns e.g. particular practical/policy goals, or a need to explore a research model in a guided way with appropriate visualisations
- Easy to use - can provide a user model allowing users to easily save, manage and share their data without needing direct access to large datasets required for code execution
- Fast - can give researchers quick means to assess utility of a model before putting in effort to download, configure and run locally
```

Reply:
Yeah, fair point, I added the prefix without due care and attention!